subprojects/matrix-benchmarking-workloads/rhods-notebooks-ux/store.py

```diff
@@ -846,6 +850,24 @@ def _parse_directory(fn_add_to_matrix, dirname, import_settings):
 
     print("parsing done :)")
 
+def _parse_start_end_times(dirname):
```

Comment:
can you add this annotation `@ignore_file_not_found` to avoid crashing the parser if the file isn't available?